prefect_aws/credentials.py

```diff
@@ -231,3 +241,12 @@ def get_s3_client(self) -> S3Client:
             An authenticated S3 client.
         """
         return self.get_client(client_type=ClientType.S3)
+
+    def get_secrets_manager_client(self) -> SecretsManagerClient:
+        """
+        Gets an authenticated Secrets Manager client.
+
+        Returns:
+            An authenticated Secrets Manager client.
+        """
+        return self.get_client(client_type=ClientType.SECRETS_MANAGER)
```

Comment:
I don't think that MinIO implements a secret manager interface, just a storage one, so `MinIOCredentials` shouldn't be able to provide a secrets manager client.
```suggestion

    def get_secrets_manager_client(self) -> SecretsManagerClient:
        """
        Gets an authenticated Secrets Manager client.

        Returns:
            An authenticated Secrets Manager client.
        """
        return self.get_client(client_type=ClientType.SECRETS_MANAGER)
```